aws/cloudformation/standalone/gen_ai_curriculum/config.rb

```diff
@@ -0,0 +1,66 @@
+require "ostruct"
+require_relative '../../../../lib/cdo/shared_constants'
+
+module Config
+  # Configuration for each endpoint used in the Gen AI Curriculum. Required properties:
+  #   hf_model_id: HuggingFace model ID
+  #   model_id: App-wide model ID (from shared constants)
+  #   model_name: Short name for naming resources (letters only)
+  #   instance_type: EC2 Instance Type for endpoint
+  #   min_num_instances (prod only): Minimum number of deployed instances used for autoscaling on production
+  #     on test, only a single instance will be used without autoscaling to limit resource usage
+  #   max_num_instances (prod only): Maximum number of deployed instances used for autoscaling on production
+  #     not used on test
+  #   autoscaling_target_value (prod only): Number of requests / minute at which autoscaling starts to take effect
+  #     not used on test
+  ENDPOINT_CONFIGS = [
+    {
+      hf_model_id: "mistralai/Mistral-7B-Instruct-v0.1",
+      model_id: SharedConstants::AI_CHAT_MODEL_IDS[:MISTRAL],
+      model_name: "Mistral",
+      instance_type: {
+        prod: "ml.g5.4xlarge",
+        test: "ml.g5.xlarge"
+      },
+      min_num_instances: 3,
+      max_num_instances: 4,
+      autoscaling_target_value: 150
+    },
+    {
+      hf_model_id: "upaya07/Arithmo2-Mistral-7B",
+      model_id: SharedConstants::AI_CHAT_MODEL_IDS[:ARITHMO],
+      model_name: "Arithmo",
+      instance_type: {
+        prod: "ml.g5.xlarge",
+        test: "ml.g5.xlarge"
+      },
+      min_num_instances: 2,
+      max_num_instances: 2,
```

Comment:
Do we possibly want to use less in test? Maybe not worth the work, but if we have any budgetary concerns, I'd love to shift more iron toward prod.

Reply:
Yep! These numbers only apply to prod - on test we will use 1 instance per endpoint with no autoscaling.